src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -113,8 +131,13 @@ public static Elevator getInstance() {
 
     @Override
     public void periodic() {
-        board.addNum("pose", getPosition());
+        board.addNum("slev1 pose", getPosition());
+        board.addNum("slev2 pose", slave2.getEncoder().getPosition());
```

Comment:
I really don't like those names but...
```suggestion
        board.addNum("slave1 pose", getPosition());
        board.addNum("slave2 pose", slave2.getEncoder().getPosition());
```